### PR TITLE
Add ViRMEN interface

### PR DESCRIPTION
**Pull request by weiglszonja**
https://github.com/catalystneuro/ndx-pinto-metadata/pull/1

### CODE REVIEW

**Comment by weiglszonja** [comment]
TODO:
- [ ] Collect synchronised timestamps for behavior data
- [ ] Create new extension for storing Pinto specific ViRMEN experiment data

**Comment by weiglszonja** [comment]
Current prototype files are in `pinto-lab-nwb/behavior/`:

- "licks" is currently added as `TimeSeries`, here is a snippet how it looks like:
licks
for those sessions where data is available:
- "optoVoltageOut" is currently added `TimeSeries`, here is a snippet how it looks like:
optoVoltageOut
- "widefield", "twop" and "eyeCam" are added as `Events` after using `get_rising_frames_from_ttl` to determine the times when they were turned on, here is a snippet how these signals looks like before using `get_rising_frames_from_ttl`:

eyecam
twop
widefield

@CodyCBakerPhD let me know if other data types would be more appropriate for these.

**Comment by CodyCBakerPhD** [comment]
Status update: just waiting for a fully converted session to confirm the synchronization

File: src/pinto_lab_to_nwb/behavior/interfaces/virmentemporalalignmentbehaviorinterface.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        reference_frame = "unknown"
+        if "SpatialSeries" in position_obj.spatial_series:
+            reference_frame = position_obj.spatial_series["SpatialSeries"].reference_frame
+        spatial_series_name = "SpatialSeriesByImFrame"
+        position_obj.create_spatial_series(
+            name=spatial_series_name,
+            data=H5DataIO(position[:, :-1], compression="gzip"),
+            description="The average x, y position by imaging frame.",
+            reference_frame=reference_frame,
```
Comment (weiglszonja) [inline src/pinto_lab_to_nwb/behavior/interfaces/virmentemporalalignmentbehaviorinterface.py:88]: add conversion here as well 
```suggestion
            reference_frame=reference_frame,
            conversion=0.01,
```

**Comment by CodyCBakerPhD** [comment]
Or are we still waiting for a full version of `sub-NCCR32` for this?

**Comment by weiglszonja** [comment]
`sub-NCCR32` should be good to review the position, velocity etc., but NCCR32 ViRMEN file doesn't have trials
`sub-Cherry` is the most complete example with the time alignment, I rerun the conversion from a fresh environment and it is being uploaded now (finishes by tomorrow morning)

**Comment by weiglszonja** [comment]
@CodyCBakerPhD No worries, I've been dealing these all the time when I was branching off to have a complete conversion branch with all the changes from these :D

**Comment by weiglszonja** [comment]
@CodyCBakerPhD after this is merged, is the current look good enough for first release `pinto-lab-to-nwb`, or should I first address all follow-up issues first?

**Comment by CodyCBakerPhD** [comment]
Good question; the biggest concern would be `ndx-holographic-stimulation`, which I don't think has been officially released yet in any capacity

Technically an official release of this ought only to rely on other pip packages instead of relying on a git install to a specific commit there, but I also don't know how much longer that is going to be

**Comment by weiglszonja** [comment]
Right, I'll keep an eye on `ndx-holographic-stimulation` and modify the README.md to install `pinto-lab-to-nwb` from Github for now

**Comment by weiglszonja** [comment]
@CodyCBakerPhD updated `sub-Cherry`
https://flatironinstitute.github.io/neurosift/?p=/nwb&url=https://api-staging.dandiarchive.org/api/assets/51d28a34-facc-48d7-b74f-4f44cec7d387/download/&dandisetId=210944&dandisetVersion=draft

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**